crates/sui-rpc-loadgen/README.md

```diff
@@ -35,4 +37,16 @@ Both commands achieve the same outcome: fetching all checkpoints using one threa
 ```bash
 cargo run --bin sui-rpc-loadgen -- --urls "http://127.0.0.1:9000" --num-threads 1 pay-sui --repeat 100
 ```
-**NOTE**: right now `pay-sui` only supports 1 thread but multi-threading support can be added pretty easily by assigning different gas coins to different threads
\ No newline at end of file
+**NOTE**: right now `pay-sui` only supports 1 thread but multi-threading support can be added pretty easily by assigning different gas coins to different threads
+
+
+### Example 3: Get All Transactions
+Finds all checkpoints, and then all transactions in those checkpoints
+
+
+## Adding other endpoints
+
```

Comment:
Thanks for adding this! After your PR is merged, we can also add link to this PR as an example